pyseed/seed_client.py

```diff
@@ -1177,6 +1178,7 @@ def upload_and_match_datafile(
         column_mapping_profile_name: str,
         column_mappings_file: str,
         import_meters_if_exist: bool = False,
+        multiple_cycle_upload: bool = False,
         **kwargs,
     ) -> dict:
         """Upload a file to the cycle_id that is defined in the constructor. This carries the
```

Comment:
can you add a comment (around line 1193) for the new 'multiple_cycle_upload' argument?